### ISSUE TITLE
CT: Assertion failed: there should only be 1 live bunny

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/natural-selection#212]
This happened only once, and is not present at any other time in CT history that goes back to 8/25/2020 6:41PM.

```
natural-selection : phet-io-mirror-inputs-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/phet-io-wrappers/mirror-inputs/?sim=natural-selection&phetioDebug&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22natural-selection%22%2C%22phet-io-mirror-inputs-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1598566856701%22%2C%22timestamp%22%3A1598568010499%7D
Uncaught Error: Uncaught Error: Assertion failed: there should only be 1 live bunny
Error: Assertion failed: there should only be 1 live bunny
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/assert/js/assert.js:22:13)
at LabModel.addAMate (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/natural-selection/js/common/model/NaturalSelectionModel.js:318:15)
at Object.addAMate (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/natural-selection/js/common/view/NaturalSelectionScreenView.js:206:31)
at listener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/natural-selection/js/common/view/PlayButtonGroup.js:47:17)
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/axon/js/TinyEmitter.js:71:18)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/axon/js/Emitter.js:33:29
at Emitter.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/axon/js/Action.js:225:18)
at Emitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/axon/js/Emitter.js:58:19)
at PushButtonModel.fire (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/sun/js/buttons/PushButtonModel.js:158:23)
at downPropertyObserver (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598566856701/sun/js/buttons/PushButtonModel.js:90:14)
id: Bayes Chrome
Snapshot from 8/27/2020, 4:20:56 PM
```

**Comment by pixelzoom** [comment]
This is almost certainly a regression introduced in 645d60d for #166. And it may even be a duplicate of #211.

**Comment by pixelzoom** [comment]
This happened once on 8/30 and once on 8/31.  So it's intermittent but should be addressed.

**Comment by pixelzoom** [comment]
I tried reproducing the `phet-io-mirror-inputs-fuzz : unbuilt` test, but discovered that fuzz is broken for mirror-inputs. See https://github.com/phetsims/phet-io-wrappers/issues/371.   And every previous occurrence of this error was in that test.

@samreid pointed out that the latest occurrence of this error is in test `fuzz : unbuilt`, so I can ignore mirror-inputs for investigation of this issue. 

```
natural-selection : fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/natural-selection/natural-selection_en.html?continuousTest=%7B%22test%22%3A%5B%22natural-selection%22%2C%22fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1598921839559%22%2C%22timestamp%22%3A1598957459544%7D&brand=phet&ea&fuzz&memoryLimit=1000
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: there should only be 1 live bunny
Error: Assertion failed: there should only be 1 live bunny
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/assert/js/assert.js:22:13)
at IntroModel.addAMate (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/natural-selection/js/common/model/NaturalSelectionModel.js:318:15)
at Object.addAMate (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/natural-selection/js/common/view/NaturalSelectionScreenView.js:206:31)
at listener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/natural-selection/js/common/view/PlayButtonGroup.js:47:17)
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/axon/js/TinyEmitter.js:71:18)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/axon/js/Emitter.js:35:29
at Emitter.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/axon/js/Action.js:226:18)
at Emitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/axon/js/Emitter.js:60:19)
at PushButtonModel.fire (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/sun/js/buttons/PushButtonModel.js:158:23)
at downPropertyObserver (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1598921839559/sun/js/buttons/PushButtonModel.js:90:14)
id: Bayes Chrome
Snapshot from 8/31/2020, 6:57:19 PM
```

**Comment by pixelzoom** [comment]
I never successfully reproduced this in my local copy - it's too intermittent. I suspect that this may have been resolved by the above commit, which reverts the change to PlayButtonGroup.js for #217.   I'll leave this open for a couple of days to see what CT has to say.

**Comment by pixelzoom** [comment]
CT indicates that this issue is fixed. Closing.
